.mailmap

```diff
@@ -28,7 +28,7 @@ Antonio D. Córcoles-Gonzales <adcorcol@us.ibm.com>
 Antonio Mezzacapo <30698465+antoniomezzacapo@users.noreply.github.com> <amezzac@us.ibm.com>
 Carmen Recio Valcarce <car_rv@outlook.es> <49230820+carmenrv@users.noreply.github.com>
 Chris Culver <chris.cb.culver@gmail.com>
-Christian Claus <cclauss@bluewin.ch>
+Christian Clauss <cclauss@me.com> <cclauss@bluewin.ch>
```

Comment:
```suggestion
Christian Clauss <cclauss@me.com>
```

Reply:
Please

Reply:
This is actually necessary, this is to map the old entries in the git log with `cclauss@bluewin.ch` to use your current email address and standardize your real name to be the same between the old commits and the current one.